### PR TITLE
Disabling viewpoint

### PR DESCRIPTION
**Pull request by CFarcy**

### CODE REVIEW

File: src/modules/RA/Viewpoint/components/ViewpointFields.js [hunk 1]
```diff
@@ -149,6 +149,7 @@ const ViewpointFields = ({
       {...props}
     >
       <FormTab label="resources.viewpoint.tabs.general">
+        <BooleanInput source="active" />
```
Comment (jrmi) [inline src/modules/RA/Viewpoint/components/ViewpointFields.js:152]: Is it easy to add something to show that the viewpoint is not visible in the list display ?

Comment (jrmi) [reply src/modules/RA/Viewpoint/components/ViewpointFields.js:152]: Like a stroked eye icon on top of the image

Comment (CFarcy) [reply src/modules/RA/Viewpoint/components/ViewpointFields.js:152]: Yes, good idea.

Comment (CFarcy) [reply src/modules/RA/Viewpoint/components/ViewpointFields.js:152]: Done.

**Review by jrmi — APPROVED** [review]

**Pull request merged**